libs/langgraph/langgraph/types.py

```diff
@@ -101,7 +101,10 @@ def default_retry_on(exc: Exception) -> bool:
 
 
 class RetryPolicy(NamedTuple):
-    """Configuration for retrying nodes."""
+    """Configuration for retrying nodes.
+    
+    !!! version-added "Added in version 0.2.24."
```

Comment:
Is there anyway to have "Added in version" be prepended automatically, so that we only put the version?